package.json

```
@@ -35,29 +35,29 @@
 	"devDependencies": {
 		"@commitlint/cli": "^12.1.4",
 		"@commitlint/config-conventional": "^12.1.4",
-		"@sapphire/eslint-config": "^3.2.1",
+		"@sapphire/eslint-config": "^3.2.2",
 		"@sapphire/prettier-config": "^1.1.6",
-		"@sapphire/ts-config": "^2.3.1",
-		"@types/node": "^15.12.4",
-		"@types/ws": "^7.4.5",
-		"@typescript-eslint/eslint-plugin": "^4.27.0",
-		"@typescript-eslint/parser": "^4.28.0",
+		"@sapphire/ts-config": "^2.3.2",
+		"@types/node": "^16.3.3",
+		"@types/ws": "^7.4.6",
+		"@typescript-eslint/eslint-plugin": "^4.28.3",
+		"@typescript-eslint/parser": "^4.28.3",
 		"cz-conventional-changelog": "^3.3.0",
 		"discord-api-types": "^0.18.1",
-		"discord.js": "^12.5.3",
+		"discord.js": "^13.0.0-dev.5ca97c9.1626480265",
```

```suggestion
		"discord.js": "dev",
```

don't we just want the latest @dev tag?